README.md

```
@@ -123,11 +231,52 @@ Check out these related projects.
-- [Reference Architectures](https://cloudposse.com/) - Launch effortlessly with our turnkey reference architectures, built either by your team or ours.
-
-
-
+- [VPA](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler) - Kubernetes Vertical Pod Autoscaler
+- [VPA Helm Chart](https://github.com/FairwindsOps/charts/tree/master/stable/vpa) - Fairwinds VPA Helm Chart
+
+
+
+
+> [!TIP]
+> #### Use Terraform Reference Architectures for AWS
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)